assignments/prototype-refactor.js

```diff
@@ -36,6 +36,18 @@ GameObject.prototype.destroy = function () {
   return `${this.name} was removed from the game.`;
 };
 */
```

Comment:
I ran this through the console and it complained about an *. It's probably somewhere in the comments. A stray asterisk that isn't being parsed correctly. I deleted all the comments and it ran without any issue.